example/integration_test/get_user_status_test.dart

```diff
@@ -0,0 +1,155 @@
+import 'package:didomi_sdk/didomi_sdk.dart';
+import 'package:didomi_sdk/events/event_listener.dart';
+import 'package:didomi_sdk_example/testapps/sample_for_get_user_status_tests.dart' as app;
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:integration_test/integration_test.dart';
+import 'util/assertion_helper.dart';
+import 'util/initialize_helper.dart';
+
+void main() {
+  IntegrationTestWidgetsFlutterBinding.ensureInitialized();
+  final initializeBtnFinder = find.byKey(Key("initializeSmall"));
+
+  bool isError = false;
+  bool isReady = false;
+
+  final listener = EventListener();
+  listener.onError = (String message) {
+    isError = true;
+  };
+  listener.onReady = () {
+    isReady = true;
+  };
+
+  DidomiSdk.addEventListener(listener);
+
+  group("User Status information", () {
+    testWidgets("Get user status without initialization", (WidgetTester tester) async {
+      // Start app
+      app.main();
+      await tester.pumpAndSettle();
+
+      assert(isError == false);
+      assert(isReady == false);
+
+      var exceptionThrown = false;
+      try {
+        await DidomiSdk.userStatus;
+      } on PlatformException {
+        exceptionThrown = true;
+      }
+      assert(exceptionThrown);
+
+      assert(isError == false);
+      assert(isReady == false);
+    });
+
+    testWidgets("Get user status with initialization", (WidgetTester tester) async {
+      // Start app
+      app.main();
+      await tester.pumpAndSettle();
+
+      await InitializeHelper.initialize(tester, initializeBtnFinder);
+
+      assert(isError == false);
+      assert(isReady == true);
+
+      var result = await DidomiSdk.userStatus;
+      assertMatchesUuidPattern(result.userId);
+      assertTextNotEmpty(result.consentString);
+      assertMatchesDatePattern(result.created);
+      assertMatchesDatePattern(result.updated);
+
+      assertListEmpty(result.purposes?.consent?.enabled);
```

Comment:
would it make sense to put all these assertListEmpty in a single method? something like assertUserStatusIsEmpty?

Reply:
I don't think so, we should only need to assert it one time.